src/Experiment.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Should the folder structure be closer to the runtime folder structure?

-src 
    - System.Experiment
        - src /
            - System.Experiment.csproj
            - MyClass.cs
        -tests /

Reply:
No, we should not have to src directories. If we change it, it should be src/Experiment/Experiment.csproj

Reply:
We should at least establish a structure that supports more than a single library. While some experiments will only need a single library, I'm sure there will be plenty of experiments that will need multiple. (We need multiple in the DataFrame library)

Reply:
Is there a specific reason to not have a tests folder per leaf?

Reply:
Either way I'm fine. I'll change it to support multiple projects per experiment then.